### PR TITLE
Added getVolume and test

### LINKED ISSUE #43: Get Volume

### PR DESCRIPTION
**Pull request by Pittsy24**
closes #43 
Signed-off-by: Joe Pitts <joepittsy@gmail.com>

### CODE REVIEW

File: KodiController/test/controller.js [hunk 1]
```diff
@@ -33,6 +41,21 @@ describe("Controller", () => {
...
 
+	describe("getVolume", () => {
+		it("should return the volume of the player", (done) => {
+			let c = new Controller();
+
+			SetNockVolume();
+
+			c.getVolume((err, volume) => {
+				if (err || !volume) done(err);
+				console.log("THE VOLUME IS", err);
```
Comment (meharryp) [inline KodiController/test/controller.js]: should be `console.log("THE VOLUME IS", volume);`,  or just delete that line

**Comment by Pittsy24** [comment]
@meharryp Could you also look into automaticly publishing the new version to NPM

File: KodiController/test/controller.js (hunk 1 above)
Comment (Pittsy24) [reply KodiController/test/controller.js]: Good catch, removed

**Comment by Pittsy24** [comment]
> @meharryp Could you also look into automaticly publishing the new version to NPM

Nvm got it

**Comment by Pittsy24** [comment]
@meharryp Please squash and merge

**Pull request merged**